keras-dreambooth-sprint/README.md

```diff
@@ -77,23 +79,31 @@ Then, we will fill the markdown part. Hyperparameters and plot is automatically
 
 You can find the example repository below 👇 
 
-[keras-dreambooth/dreambooth_diffusion_model · Hugging Face](https://huggingface.co/keras-dreambooth/dreambooth_diffusion_model)
+[keras-dreambooth/dreambooth_diffusion_model](https://huggingface.co/keras-dreambooth/dreambooth_diffusion_model)
 
 ### Model Demo
 
 We will use Gradio to build our demos for the models we have trained. With `Interface` class it’s straightforward 👇  
 
 ```python
+from huggingface_hub import from_pretrained_keras
+from keras_cv import models
 import gradio as gr
 
-# write function for inference
+sd_dreambooth_model = models.StableDiffusion(
+    img_width=512, img_height=512
+)
+db_diffusion_model = from_pretrained_keras("merve/dreambooth_diffusion_model")
+sd_dreambooth_model._diffusion_model = db_diffusion_model
+
+# generate images
 def infer(prompt):
-# assume you loaded the model
     generated_images = sd_dreambooth_model.text_to_image(
-        prompt, batch_size=4
+        prompt, batch_size=2
```

Comment:
Let's parameterize the `infer()` method with another argument called `num_images_to_gen` and set that as the `batch_size` value. The `batch_size` argument in `text_to_image()` is a bit confusing.

Reply:
I really want to keep this simple and more as a prior to gradio, does it work? I will remove batch size.